docs/data-sources/equinix_fabric_connection.md

```diff
@@ -0,0 +1,418 @@
+
+
+<a id="nestedobjatt--a_side--access_point--gateway"></a>
+### Nested Schema for `a_side.access_point.gateway`
+
+Read-Only:
+
+- `href` (String)
+- `state` (String)
+- `uuid` (String)
+
+
+<a id="nestedobjatt--a_side--access_point--interface"></a>
+### Nested Schema for `a_side.access_point.interface`
+
+Read-Only:
+
+- `id` (String)
+- `type` (String)
+- `uuid` (String)
+
+
+<a id="nestedobjatt--a_side--access_point--link_protocol"></a>
+### Nested Schema for `a_side.access_point.link_protocol`
+
+Read-Only:
+
+- `type` (String)
+- `vlan_c_tag` (Number)
+- `vlan_s_tag` (Number)
+- `vlan_tag` (Number)
+
+<a id="nestedobjatt--a_side--access_point--location"></a>
+### Nested Schema for `a_side.access_point.location`
+
+Read-Only:
+
+- `ibx` (String)
+- `metro_code` (String)
+- `metro_name` (String)
+- `region` (String)
+
+
+<a id="nestedobjatt--a_side--access_point--port"></a>
+### Nested Schema for `a_side.access_point.port`
+
+Read-Only:
+
+- `href` (String)
+- `name` (String)
+- `redundancy` (Set of Object) (see [below for nested schema](#nestedobjatt--a_side--access_point--port--redundancy))
+- `uuid` (String)
+
+<a id="nestedobjatt--a_side--access_point--port--redundancy"></a>
+### Nested Schema for `a_side.access_point.port.uuid`
+
+Read-Only:
+
+- `priority` (String)
+
+
+
+<a id="nestedobjatt--a_side--access_point--profile"></a>
+### Nested Schema for `a_side.access_point.profile`
+
+Read-Only:
+
+- `description` (String)
+- `href` (String)
+- `name` (String)
+- `type` (String)
+- `uuid` (String)
+
+
+<a id="nestedobjatt--a_side--access_point--virtual_device"></a>
+### Nested Schema for `a_side.access_point.virtual_device`
+
+Read-Only:
+
+- `href` (String)
+- `type` (String)
+- `uuid` (String)
+
+
+
+<a id="nestedatt--a_side--service_token"></a>
+### Nested Schema for `a_side.service_token`
+
+Read-Only:
+
+- `description` (String)
+- `href` (String)
+- `type` (String)
+- `uuid` (String)
+
+
+
+<a id="nestedblock--notifications"></a>
+### Nested Schema for `notifications`
+
+Read-Only:
+
+- `emails` (List of String) Array of contact emails
+- `send_interval` (String) Send interval
+- `type` (String) Notification Type
+
+
+<a id="nestedblock--z_side"></a>
+### Nested Schema for `z_side`
+
+Read-Only:
+
+- `access_point` (Set of Object) Point of access details (see [below for nested schema](#nestedatt--z_side--access_point))
+- `service_token` (Set of Object) For service token based connections, Service tokens authorize users to access protected resources and services. Resource owners can distribute the tokens to trusted partners and vendors, allowing selected third parties to work directly with Equinix network assets (see [below for nested schema](#nestedatt--z_side--service_token))
+
+<a id="nestedatt--z_side--access_point"></a>
+### Nested Schema for `z_side.access_point`
+
+Read-Only:
+
+- `account` (Set of Object) (see [below for nested schema](#nestedobjatt--z_side--access_point--account))
+- `authentication_key` (String)
```

Comment:
There's a lot of arguments without description in the docs but I see the description below in the schema. Since you're using generated docs, maybe it needs just a refresh?

Reply:
I do recall some limitations in the docs generation process.
These come to mind: https://github.com/hashicorp/terraform-plugin-docs/issues?q=is%3Aissue+is%3Aopen+nested

Reply:
We will want to see the description copied in from the schema to each of these blank fields. The doc generator gave us a headstart, but we'll want to avoid using it until these upstream problems have been addressed in the generator.

Reply:
Generally if the names are self explanatory, I think detailed description is ignored in the spec, but we need to inform team.